docs/Configuration.md

```diff
@@ -44,6 +44,7 @@ You can add these configurations into spark-defaults.conf to enable or disable t
 | spark.gluten.sql.columnar.wholeStage.fallback.threshold | Configure the threshold for whether whole stage will fall back in AQE supported case by counting the number of ColumnarToRow & vanilla leaf node                                                                                                                                                                                                                                                                                                     | \>= 3                                                |
 | spark.gluten.sql.columnar.query.fallback.threshold      | Configure the threshold for whether query will fall back by counting the number of ColumnarToRow & vanilla leaf node                                                                                                                                                                                                                                                                                                                                 | \>= 1                                                |
 | spark.gluten.sql.columnar.maxBatchSize                  | Set the number of rows for the output batch                                                                                                                                                                                                                                                                                                                                                                                                          | 4096                                                 |
+| spark.gluten.shuffleWriterBufferSize                  | Set the number of buffer rows for the shuffle writer                                                                                                                                                                                                                                                                                                                                                                                                          | 4096                                                 |
```

Comment:
nit: ```spark.gluten.shuffleWriter.BufferSize``` better? Similar name convention to ```spark.shuffle.file.buffer``` in Spark.

Reply:
OK